data/scenarios/World Examples/clearing.yaml

```
@@ -14,9 +14,16 @@ robots:
     dir: north
 world:
   dsl: |
-    overlay
+    let // Stretch the y coordinates used for the radius calculation
```

```suggestion
    let // Weight the y coordinates used for the radius calculation
```

How about "weight the y-coordinates"?